psa-crypto/src/types/algorithm.rs

```diff
@@ -947,6 +987,13 @@ impl From<RawKeyAgreement> for psa_crypto_sys::psa_algorithm_t {
     }
 }
 
+#[cfg(feature = "interface")]
```

Comment:
Does it need this feature?

Reply:
Nope, good spot. This has already been added in #51 without it so I can remove it here and rebase onto master.